README.md

```diff
@@ -428,12 +428,24 @@ good
 
 bad
   .chain(eitherToMaybe) // Nothing
+
+// Transformations from array and back
+const listFromArray =
+  List.fromArray([1,2,3]) // List [ 1, 2, 3 ]
+
+const arrayFromList =
+  listToArray(listFromArray) // [ 1, 2, 3 ]
+
+const andBackAgain =
+  arrayToList(arrayFromList) // List [ 1, 2, 3 ]
 ```
 
 #### Transformation Signatures
 | Transform | ADT signature | Function Signature |
 |---|---|---|
+| `arrayToList` | `a -> List a` | `(a -> b) -> a -> b` |
```

Comment:
This should really be something like:
* ADT: `[ a ] -> List a`.
* Function: `(a -> [ b ]) -> List b`